src/xenia/kernel/user_module.cc

```diff
@@ -135,7 +135,7 @@ X_STATUS UserModule::LoadFromMemory(const void* addr, const size_t length) {
   auto processor = kernel_state()->processor();
 
   auto magic = xe::load_and_swap<uint32_t>(addr);
-  if (magic == 'XEX2') {
+  if (magic == 'XEX2' || magic == 'XEX1') {
```

Comment:
Do you think it makes sense to compare just first three characters, aka `XEX` here? (with an appropriate mask)

Reply:
I had asked myself the same question but as I saw that it was comparing with `XEX2` I wasn't sure how specific it should be. There are actually a few more unhandled magics such as `XEX%`, `XEX-`, and `XEX?`.

Reply:
I actually wonder then if those are still 4-byte magics or 3-byte magics with some other data coming after them - but having unaligned data would be quite weird IMO.